policyDefinitions/General/deploy-resource-lock-on-rgs-tag-exclusion/azurepolicy.json

```diff
@@ -5,7 +5,7 @@
     "displayName": "Deploy Resource Lock on RGs - tag exclusion",
     "description": "Automatically deploys a resource lock to resources that are not in the dv environment to prevent accidental deletion.",
```

Comment:
typo: 'dv' environment?

Reply:
@DonKoning That's a good question. I assume this is merely an example of a tag value for exclusion.
It could be a typo, but it appears intentional since the same value is used in two different places.
So, should we change it or leave it as it is?

Reply:
My suggestion is to change it to 'dev'. If it's not clear to us, than probably it will not be clear to others either.

Reply:
@DonKoning Changed the value to 'dev'. Is this ok with you?

Reply:
Absolutely